torch/_inductor/codegen/cuda/cuda_scheduling.py

```diff
@@ -0,0 +1,46 @@
+from typing import List
+
+from ... import config
+from ...codecache import code_hash, get_path
```

Comment:
It would be better to use absolute imports here, as that eases code navigation tools ( both in IDEs and github code views for example )